init.fish

```diff
@@ -26,6 +26,11 @@ else
   set -gx ORIGINAL_PATH $PATH
 end
 
+# Set OMF_CONFIG if not overriden.
+if not set -q OMF_CONFIG
+  set -gx OMF_CONFIG "$HOME/.config/omf"
```

Comment:
nope. we need to respect XDG, not just support its default value.

we need to replicate this (from the current install script):

```
 test -z ${XDG_CONFIG_HOME+_}  && XDG_CONFIG_HOME="${HOME}/.config"
 test -z ${OMF_CONFIG+_}       && OMF_CONFIG="${XDG_CONFIG_HOME}/omf"
```